### PR TITLE
Authentication and Landing pages

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
### Files to check:
- LandingPage.jsx
- Login.jsx
- Signup.jsx
- auth-context.jsx
- CSS files

### Files to ignore:
- Any other files with minor modifications

#### Landing Page
image

### Login Page
image

### Signup Page
image

### CODE REVIEW

**Review by foolhardy21 — APPROVED** [review]
Code looks good. Small detail that i liked is that you have destructured the response right away.

File: src/pages/auth/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        <p>
+          <small>
+            By continuing, you agree to Dazzle Cerashop's
+            <span className="secondary-text fw-700">Terms of use</span> and
+            <span className="secondary-text fw-700">Privacy Policy.</span>
+          </small>
+        </p>
+        <button
+          className="button primary form-btn mt-0p5"
+          onClick={() => signUpHandler()}
```
Comment (mikesch803) [inline src/pages/auth/Signup.jsx:114]: here you are not passing e

File: src/pages/auth/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        <h4 className="text-center pt-1 pb-2 m-0">Login</h4>
+        <div className="input-field mb-2">
+          <input
+            className="input"
+            name="email"
+            type="email"
+            required=""
+            value={formData.email}
+            onChange={(e) =>
+              setFormData((form) => ({ ...form, email: e.target.value }))
```
Comment (mikesch803) [inline src/pages/auth/Login.jsx:50]: suggestion : you are using name attribute you can also do like [e.target.name]:e.target.value in every input field

File: src/pages/auth/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import "./auth.css";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../context";
+
+const Login = () => {
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+  });
```
Comment (mikesch803) [inline src/pages/auth/Login.jsx:10]: suggestion : here you can pass only empty obj as you are using name attribute

**Review by mikesch803 — APPROVED** [review]
your code is good to merge, i didn't find much

File: src/pages/auth/Login.jsx (hunk 3 above)
Comment (ApurvaSawant11) [reply src/pages/auth/Login.jsx:10]: Will try to make the changes in the next PR  while implementing auth routes. Thanks for the suggestion

**Pull request merged**